kart/lfs_util.py

```diff
@@ -49,6 +78,26 @@ def pointer_file_to_json(pointer_file_bytes, result=None):
     return result
 
 
+def dict_to_pointer_file_bytes(pointer_dict):
+    def sort_key(key_value):
+        key, value = key_value
+        if key == "version":
+            return ""
+        return key
+
+    blob = bytearray()
+    for key, value in sorted(pointer_dict.items(), key=sort_key):
+        # TODO - LFS doesn't support our fancy pointer files yet. Hopefully fix this in LFS.
+        if key not in ("version", "oid", "size"):
+            continue
+        blob += key.encode("utf8")
+        blob += b" "
+        blob += str(value).encode("utf8")
+        blob += b"\n"
+
+    return blob
```

Comment:
```suggestion
    blob = bytearray()
    for key, value in sorted(pointer_dict.items(), key=lambda i: (i[0] != "version", i)):
        # TODO - LFS doesn't support our fancy pointer files yet. Hopefully fix this in LFS.
        if key not in ("version", "oid", "size"):
            continue
        blob += f"{key} {value}\n".encode("utf8")

    return blob
```

Reply:
> `if key not in ("version", "oid", "size"):`

This ignores all the `ext-*` ones, but I guess for now it's ok to ignore them? Nothing else will be writing Kart repos

Reply:
Yeah - this is a temporary workaround anyway. Plenty of time to fix it if we start writing extensions into pointer files